morf-core/src/main/java/org/alfasoftware/morf/sql/NoDirectPathQueryHint.java

```diff
@@ -0,0 +1,16 @@
+package org.alfasoftware.morf.sql;
+
+/**
+ * Singleton class representing the hint for avoiding using direct path query.
+ *
+ * @author Copyright (c) Alfa Financial Software Limited. 2022
+ */
+public final class NoDirectPathQueryHint implements Hint {
+
+  private NoDirectPathQueryHint() {
+    super();
+  }
+
+  public static final NoDirectPathQueryHint INSTANCE = new NoDirectPathQueryHint();
+
+}
```

Comment:
This was designed to add NOAPPEND hint, the implementation of this is not yet finished under this PR.
See MORF-41 for details, which suggests the same.